server/lib/src/server/migrations.rs

```diff
@@ -895,6 +897,23 @@ impl<'a> QueryServerWriteTransaction<'a> {
                 err
             })?;
 
+        // all the built-in objects get a builtin class
+        let filter = f_and(vec![
+            f_lt(
+                Attribute::Uuid,
+                PartialValue::Uuid(uuid::uuid!("00000000-0000-0000-0001-000000000000")),
```

Comment:
uuid as constant for "START_OF_DYNAMIC_RANGE" or something like that?

Reply:
`DYNAMIC_RANGE_MINIMUM_UUID` done!